README.md

```diff
@@ -131,29 +131,34 @@ The following parameters are supported:
+* **upload.expiration:** The time an interrupted upload is stored for continuation in the future in milliseconds. If this time expires, the upload must start from the beginning.
 * **measurement.payload.limit:** The size of a measurement in bytes up to which it is accepted as a single upload. Larger measurements are transmitted in chunks.
-* **storage-type:** The type of storage to use for the uploaded data. Currently either `gridfs` or `google` is supported. The following parameter are required:
+* **storage-type:** The type of storage to use for the uploaded data. Currently, either `gridfs` or `google` is supported. The following parameter are required:
   * **gridfs**
     * **type:** Must be `gridfs` in this case.
-    * **uploads-folder:** The relative or absolute path to a folder, to store temporary not finished uploads on the local harddrive before upload of the complete data blob to GridFS upon completion.
+    * **uploads-folder:** The relative or absolute path to a folder, to store temporary not finished uploads on the local hard drive before upload of the complete data blob to GridFS upon completion.
   * **google**
     * **type:** Must be `google` in this case.
-    * **collection-name:** The name of a Mongo collection to store an uploads metadata.
+    * **collection-name:** The name of a Mongo collection to store uploads' metadata into.
     * **project-identifier:** A Google Cloud Storage project identifier to where the upload bucket is located.
     * **bucket-name:** The Google Cloud Storage bucket name to load the data into.
     * **credentials-file:** A credentials file used to authenticate with the Google Cloud Storage account used to upload the data to the Cloud.
     * **paging-size:** The number of buckets to load per request, when iterating through all the data uploaded. Large numbers require fewer requests but more memory.
+* **auth-type:** The type of authentication service to use. Currently, either `mocked` or `keycloak` is supported. Defaults to `keycloak`. Both require the following parameters:
+  * **keycloak.callback**: The callback URL you entered in your provider admin console. This defaults to `http://localhost:8080/callback`.
+  * **keycloak.client**: The name of the Keycloak client to contact. This defaults to `collector`.
+  * **keycloak.secret**: The secret of the Keycloak client to contact.
+  * **keycloak.site**: The Root URL for the provider without trailing slashes. This defaults to `https://auth.cyface.de:8443/realms/{tenant}`.
+  * **keycloak.tenant**: The name of the Keycloak real to contact. This defaults to `rfr`.
```

Comment:
Actually we are using OpenID or OAuth2. Keycloak is just the implementation. I guess it would be smarter to not use the term keycloak here, but always something like openid or oauth2 so we can switch the Identity Provider without having to rename this.